react/week1/class1hw-app/src/Todolists/Todolists.jsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import Todolist from "../Todolist/Todolist";
+import "./style.css";
+
+class Todolists extends React.Component{
+    render(){
+        return <div className="todos-card">
+                <Todolist 
+                task= "Get out of bed." 
+                when= "(Wed Sep 13 2017)" 
+                banner="https://upload.wikimedia.org/wikipedia/en/thumb/f/ff/Drax_Project_featuring_Hailee_Steinfeld_-_Woke_Up_Late.png/220px-Drax_Project_featuring_Hailee_Steinfeld_-_Woke_Up_Late.png"
+                />
+                <Todolist 
+                task= "Brush teeth." 
+                when= "(Thu Sep 14 2017)" 
+                banner="https://upload.wikimedia.org/wikipedia/commons/4/49/Toothpasteonbrush.jpg"
+                />
+                <Todolist 
+                task= "Eat breakfast." 
+                when= "(Fri Sep 15 2017)" 
+                banner="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9b/NCI_Visuals_Food_Meal_Breakfast.jpg/1200px-NCI_Visuals_Food_Meal_Breakfast.jpg"
+                 />
+                </div>
+    }
```

Comment:
Here, instead of hardcodeing each todo, we can make the component loop throgh an array of todos and return the `<TodoList/>` component that you created ,fx:

```suggestion
 const todoArray = [
        {
            description: "Get out of bed",
            deadline: "Wed Sep 13 2017",
            banner:
                "https://upload.wikimedia.org/wikipedia/en/thumb/f/ff/Drax_Project_featuring_Hailee_Steinfeld_-_Woke_Up_Late.png/220px-Drax_Project_featuring_Hailee_Steinfeld_-_Woke_Up_Late.png",
        },
        ...
    ];
    return (
        <div>
            {todoArray.map(todo => (
                <Todolist
                    task={todo.description}
                    when={todo.deadline}
                    banner={todo.banner}
                />
            ))}
        </div>
    );
```

Reply:
I will do this one.Thank you.